Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/day/tabs/person/ContactDiaryPersonListAdapter.kt

```diff
@@ -22,10 +23,13 @@ internal class ContactDiaryPersonListAdapter(
 
     override fun onBindBaseVH(holder: CachedPersonViewHolder, position: Int, payloads: MutableList<Any>) {
         val item = data[position]
+        val codeThrottle = CodeThrottle()
```

Comment:
I'd prefer an extension function on View. E.g. `fun View.setOnThrottlesClicklistener()` or, alternatively, an ThrottledClickListener (overriding the regular one).

Reply:
Thinking out loud:
* `fun View.setOnClickListenerDebounced { }`
* `fun View.clicks(): Flow<View> -> View.clicks().debounced()` not sure if it would be a burden or an asset to have this as `Flow` :thinking:

Reply:
Maybe `debounced` is the wrong name, what the code currently does is `ThrottleFirst`, e.g. http://reactivex.io/documentation/operators/sample.html which seems to be what we want.

Reply:
@d4rken I implemented it as an extension function now as @ralfgehrer proposed. I didn't use `Flow` since this would add more complexity in my opinion